.moai/specs/SPEC-UPDATE-YAML-PRESERVE-001/acceptance.md

```
@@ -38,8 +39,10 @@ All commands are run from the repository root `/Users/goos/MoAI/moai-adk-go`.
 **Then** the output contains the same number of `#`-bearing lines as the input.
 
 ```bash
-go test ./internal/cli/update/backup/ -run TestPreserveGolden_Comments -count=1 -v
+go test ./internal/cli/update/backup/ -run TestPreserveGolden_Comments -count=1 -v 2>&1 \
+  | tee /dev/stderr | grep -q '^--- PASS: TestPreserveGolden_Comments'
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)